Makefile

```diff
@@ -130,16 +130,13 @@ test:
 	@echo
 	go test -tags "${NOTARY_BUILDTAGS}" $(TESTOPTS) $(PKGS)
 
-test-full: TESTOPTS =
```

Comment:
The only reason is because CI doesn't use it, and doesn't have the full linting.  I can just add those, though.